dev/io.openliberty.jakarta.data.1.0/src/jakarta/data/page/Pagination.java

```diff
@@ -60,6 +61,11 @@ public Pageable beforeKeysetCursor(Pageable.Cursor cursor) {
         return new Pagination(page, size, sorts, Mode.CURSOR_PREVIOUS, cursor);
     }
 
+    @Override
+    public Optional<Cursor> cursor() {
+        return type == null ? Optional.empty() : Optional.of(type);
```

Comment:
I'm sure the spec is defined this way, but the check is unnecessary we would have just written: 
```suggestion
        return Optional.ofNullable(type);
```